packages/web3-eth-contract/src/types.ts

```diff
@@ -227,7 +227,11 @@ export interface NonPayableMethodObject<Inputs = unknown[], Outputs = unknown[]>
 	 * @param block - If you pass this parameter it will not use the default block set with contract.defaultBlock. Pre-defined block numbers as `earliest`, `latest`, and `pending` can also be used. Useful for requesting data from or replaying transactions in past blocks.
 	 * @returns - The return value(s) of the smart contract method. If it returns a single value, it’s returned as is. If it has multiple return values they are returned as an object with properties and indices.
 	 */
-	call(tx?: NonPayableCallOptions, block?: BlockNumberOrTag): Promise<Outputs>;
+
+	call<SpecialOutput = undefined>(
```

Comment:
why not just default to Outputs?

Reply:
because later I use conditional type and check if it is undefined then use Outputs

Reply:
fixed. thanks